src/plugins/main.ts

```diff
@@ -14,6 +14,7 @@ import PluginSerializable from './plugin-serializable';
 import type {
   AnyFn,
   PartialGameState,
+  TransientState,
```

Comment:
In a subsequent change I want to add more negative cases to this test suite. The added test is the only one so far that actually exercises a reducer negative case. In fact, there's a ton of duplicated error handling in the reducer and master that could just be delegated to the reducer in the next step of this feature.